### PR TITLE
File preview and better file icons

### PR DESCRIPTION
**Pull request by apardyl**

### CODE REVIEW

File: src/main/resources/templates/layout.html [hunk 1]
```diff
@@ -36,7 +37,7 @@
                     <div class="input-group mr-sm-2">
                         <input class="form-control" type="text" placeholder="Search" aria-label="Search">
                         <div class="input-group-append">
-                            <button class="btn btn-secondary" type="submit"><i class="fas fa-search"></i></button>
+                            <button class="btn btn-secondary" type="submit"><i class="fas fa-search" ></i></button>
```
Comment (m4tx) [inline src/main/resources/templates/layout.html]: Seems like accidentally added space.
```suggestion
                            <button class="btn btn-secondary" type="submit"><i class="fas fa-search"></i></button>
```

File: src/main/resources/templates/preview_code.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    <div class="card mt-2">
+        <div class="card-header">
+            <a href="#" th:href="@{${download}}" class="btn btn-secondary"><i class="fas fa-download"></i> Download</a>
+        </div>
+        <pre class="card-body p-0 m-0"><code th:text="${text}">
+        import test
+
+        print("This is a sample code 1234")
+    </code></pre>
```
Comment (m4tx) [inline src/main/resources/templates/preview_code.html]: I think this should be indented.
```suggestion
        </code></pre>
```

File: detekt.yml [hunk 3]
```diff
@@ -75,7 +75,7 @@ complexity:
     includeStaticDeclarations: false
   ComplexMethod:
     active: true
-    threshold: 10
+    threshold: 15
```
Comment (m4tx) [inline detekt.yml]: Looks like a dirty workaround. Which method is complex that it requires this change?

Comment (apardyl) [reply detekt.yml]: `threshold: 10` is too strict (discussed in person)

**Pull request merged**